docs/vector.rst

```diff
@@ -101,6 +107,9 @@ As a temporary local variable
 Methods
 -------
 
+Vector related methods in Taichi are quite like those in GLSL (OpenGL shader language). Thus saving learning efforts.
+
+.. function:: ti.length(a, eps = 0)
```

Comment:
Again, we should follow the "There should be one-- and preferably only one --obvious way to do it." rule.

Reply:
Let's remove `a.dot(b)` then.